lib/assets/javascripts/react_ujs_mount.js

```diff
@@ -2,6 +2,31 @@
   // jQuery is optional. Use it to support legacy browsers.
   var $ = (typeof window.jQuery !== 'undefined') && window.jQuery;
 
+  // Get the constructor for a className
+  // Not a part of the public API
+  function getConstructor(className) {
```

Comment:
this function could be _really_ useful for people using NodeJS-type builds, could you make it part of `ReactRailsUJS`? 

(That way, people could override `getConstructor` to hook into their build systems!!)